toolbox/deps_update/main.go

```diff
@@ -126,15 +126,15 @@ func updateDependenciesOf(repo string) error {
 	if err != nil {
 		return err
 	}
-	repoDir, err := u.CloneRepoCheckoutBranch(githubClnt, repo, *baseBranch, "", "src/istio.io")
+	repoDir, err := u.CloneRepoCheckoutBranch(githubClnt, repo, *baseBranch, "", "code/src/istio.io")
```

Comment:
Based on what I have in my tree and what's used in green-builds manifest, I think this should you should be adding "go" rather than "code".  You'll also probably find that the other stuff you've added for creating the bin directory won't be necessary (I haven't needed to do this, and if it is necessary is seemed like a mkdir -p should be in the Makefile's rule for ${GOPATH}/bin/dep

Reply:
weird. it was not working for me before. But not it does. Changed the path to go as suggested.